src/Symfony/Component/Form/Tests/ChoiceList/Loader/AbstractChoiceLoaderTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\ChoiceList\Loader;
+
+use Symfony\Component\Form\ChoiceList\LazyChoiceList;
+
+/**
+ * @author Jules Pietri <jules@heahprod.com>
+ */
+abstract class AbstractChoiceLoaderTest extends \PHPUnit_Framework_TestCase
```

Comment:
Don't you think it might help to introduce other loaders in the future and test them ?
I did it because it's the first implementation in the form component (the only one before was the `DoctrineChoiceLoader`) but this part should be re-usable IMHO.

Of courses I can easily integrate it in `CallbackChoiceLoaderTest` class.

Is this the final blocker ? Thank you again for your review.